modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2236,7 +2239,7 @@ def getitem_column_array(self, key, numeric=False):
             new_modin_frame = self._modin_frame.take_2d_labels_or_positional(
                 col_labels=key
             )
-        return self.__constructor__(new_modin_frame)
+        return self.__constructor__(new_modin_frame, shape_hint)
```

Comment:
```suggestion
        return self.__constructor__(new_modin_frame, shape_hint=shape_hint)
```
If a parameter is one of the kwargs, I prefer to pass it as a keyword arg.

Reply:
Do you mean if the parameter is optional?

Reply:
yep